### ISSUE TITLE
npm i concurrently not found [9:58pm]

### ISSUE THREAD

**Issue by frankcollins3** [frankcollins3/fill_container#259]
**attempting to do:**
deploy with EC2, using iterm as SSH-i client instead of terminal emulator, done so with ubuntu

**error:**
ubuntu@ip:/fill_container$ cd ..
ubuntu@ip/$ sudo apt npm install -g concurrently
E: Command line option 'g' [from -g] is not understood in combination with the other options.
ubuntu@ip-:/$ apt-get install nodejs-legacy
E: Could not open lock file /var/lib/dpkg/lock-frontend - open (13: Permission denied)
E: Unable to acquire the dpkg frontend lock (/var/lib/dpkg/lock-frontend), are you root?
ubuntu@ip/$ sudo apt-get install nodejs-legacy
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Package nodejs-legacy is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'nodejs-legacy' has no installation candidate
ubuntu@ip$

**proposed approach:**
1: this site has a few terminal commands to try.
https://stackoverflow.com/questions/38343657/sh-1-concurrent-not-found-while-npm-start

**Comment by frankcollins3** [comment]
**ran npm i -g concurrently from iterm as terminal emulator from desktop scope:**
```
➜  ~ cd desktop
➜  desktop npm i -g concurrently
npm WARN checkPermissions Missing write access to /usr/local/lib/node_modules
npm ERR! code EACCES
npm ERR! syscall access
npm ERR! path /usr/local/lib/node_modules
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, access '/usr/local/lib/node_modules'
npm ERR!  [Error: EACCES: permission denied, access '/usr/local/lib/node_modules'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'access',
npm ERR!   path: '/usr/local/lib/node_modules'
npm ERR! }
npm ERR!
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR!
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/medium/.npm/_logs/2023-07-03T02_00_37_123Z-debug.log
**➜  desktop sudo npm i -g concurrently
Password:**
/usr/local/bin/concurrently -> /usr/local/lib/node_modules/concurrently/dist/bin/concurrently.js
/usr/local/bin/conc -> /usr/local/lib/node_modules/concurrently/dist/bin/concurrently.js
+ concurrently@8.2.0
added 30 packages from 30 contributors in 23.566s
➜  desktop
```
[10:01pm]

**Comment by frankcollins3** [comment]
deleted client/node_modules && server/node_modules.
package.json in [main-root, client, server] but node_modules only in main-root 
[12:00pm next day] this was solved also by solving issue #256 had an install fest for webpack because of puppeteer files.

os, browserify-os, stream, child_process, etc.
